yourbench/main.py

```diff
@@ -119,6 +120,27 @@ def gui() -> None:
     raise NotImplementedError("GUI support is not yet implemented.")
 
 
+@app.command(help="Generate a YourBench configuration file interactively.")
```

Comment:
How can you make sure this is always up to date with the modifs in the core code?